src/fetch/payouts.py

```diff
@@ -73,25 +79,27 @@ def __init__(  # pylint: disable=too-many-arguments
         solver: Address,
         solver_name: str,
         reward_target: Address,
-        bonding_pool: Address,
+        buffer_accounting_target: Address,
```

Comment:
The name might not sound self-explanatory. could you add a comment/clarification about what this is?

Reply:
I added a comment.